test/system/002-help.bats

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bats
+
+load helpers
+
+@test "Show usage screen when no command is given" {
+  run_toolbox 1
+  is "${lines[0]}" "toolbox: missing command" "Usage line 1"
+}
```

Comment:
Nitpick: some of these new files don't have a trailing newline, which Git complains about with `\ No newline at end of file`.

Reply:
Added the newlines.